pgvectorscale/src/access_method/build.rs

```diff
@@ -571,28 +571,29 @@ pub mod tests {
             ",
         ))?;
 
-        let with_index: Option<Vec<pgrx::pg_sys::ItemPointerData>> = Spi::get_one_with_args(
-            &format!(
-                "
-        SET enable_seqscan = 0;
-        SET enable_indexscan = 1;
-        SET diskann.query_search_list_size = 25;
-        WITH cte AS (
-            SELECT
-                ctid
-            FROM
-                {table_name}
-            ORDER BY
-                embedding <=> $1::vector
-            LIMIT 10
-        )
-        SELECT array_agg(ctid) from cte;"
-            ),
-            vec![(
-                pgrx::PgOid::Custom(pgrx::pg_sys::FLOAT4ARRAYOID),
-                test_vec.clone().into_datum(),
-            )],
-        )?;
+        // TODO: uncomment this once we can use https://github.com/pgcentralfoundation/pgrx/pull/1900
```

Comment:
I had to comment out this part of the test because of a problem with pgrx that should be fixed soon: https://github.com/pgcentralfoundation/pgrx/pull/1900

Reply:
Yes, I was able to stringify the ctid and all the tests still pass.